META.json

```diff
@@ -77,7 +77,7 @@
             "URI" : "1.10",
             "URI::Escape" : "0",
             "WWW::RobotRules" : "6",
-            "base" : "0",
+            "parent" : "0.217",
```

Comment:
What's your reasoning for requiring this specific version?

Reply:
Depend on the minimum version to support `-norequire`. As I noted in the commit and PR message.